discord/guild.py

```diff
@@ -750,6 +769,28 @@ def get_member(self, user_id: int, /) -> Optional[Member]:
             The member or ``None`` if not found.
         """
         return self._members.get(user_id)
+    
+    async def try_getting_member(self, user_id: int) -> Member:
```

Comment:
This is indeed a frequently requested feature, I'm glad you implemented it. Here are some comments:

- `user_id` should probably be named `member_id`
- You can use `self.get_member` instead of `self._members.get`
- You don't need `if try_member is None` since it's always true at this point of code
- `self.fetch_member` works just fine, it also returns a `Member` object
- You forgot to add the member object to the guild's cache: `self._add_member(member)`
- I suggest to name this method as `getch_member`

Reply:
Done this ^^^